code/game/machinery/doors/poddoor.dm

```diff
@@ -43,7 +43,10 @@
 
 /obj/machinery/door/poddoor/crowbar_act(mob/living/user, obj/item/tool)
 	. = ..()
-	if (density)
+	if(machine_stat & NOPOWER)
+		open(TRUE)
+		return
+	if (density && hasPower())
```

Comment:
Isn't `hasPower` just a `machine_stat & NOPOWER` check? why duplicate it? couldn't this just be a check for density?

Reply:
I think you're right, let me test.

Reply:
Yeah you're right.